### PR TITLE
updated incorrect CIS controls Ids and version mapping

### PR DESCRIPTION
**Pull request by KiranBodipi**
updated the scripts with correct CIS control Ids for controls Ids and version mapping
1. package/cfg/k3s-cis-1.20-hardened/master.yaml - 1.2.25 id missing, found duplicate id for 1.2.26
2. package/cfg/k3s-cis-1.23-permissive/master.yaml - 1.2.23 id missing, found duplicate id for 1.2.25
3. package/cfg/k3s-cis-1.24-permissive/master.yaml - 1.2.23 id missing, found duplicate id for 1.2.25
4. package/cfg/rke-cis-1.23-hardened/master.yaml - 1.2.23 id missing, found duplicate id for 1.2.25
5. package/cfg/rke-cis-1.23-permissive/etcd.yaml - incorrect version mapping found
6. package/cfg/rke-cis-1.24-hardened/master.yaml - 1.2.23 id missing, found duplicate id for 1.2.25
7. package/cfg/rke-cis-1.24-permissive/etcd.yaml - incorrect version mapping found
8. package/cfg/rke2-cis-1.23-hardened/master.yaml - 1.2.23 id missing, found duplicate id for 1.2.25
9. package/cfg/rke2-cis-1.24-hardened/master.yaml - 1.2.23 id missing, found duplicate id for 1.2.25

### CODE REVIEW

**Comment by KiranBodipi** [comment]
Hey @macedogm @rayandas Found and fixed errors in the repo. Submitted a PR—please review and merge if approved

File: package/cfg/rke-cis-1.23-permissive/etcd.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 controls:
-version: "cis-1.23"
+version: 1.23
```
Comment (andypitcher) [inline package/cfg/rke-cis-1.23-permissive/etcd.yaml:3]: @KiranBodipi regarding the version name, I agree we should stick to kube-bench's format which uses `version: 1.X `and not `version: cis-1.x`.
IMO the cis-benchmark chart (or any other components) use cfgs filenames, so setting back the naming convention to version: 1.X in the yamls should be fine.
@rayandas could you confirm this ?
If we all approve this, let's change all files to use `version: 1.X`.

**Review by andypitcher — CHANGES_REQUESTED** [review]

File: package/cfg/rke-cis-1.23-permissive/etcd.yaml (hunk 1 above)
Comment (rayandas) [reply package/cfg/rke-cis-1.23-permissive/etcd.yaml:3]: Yeah we normally follow `version: 1.x` only. But I saw for a few configs there are `version: cis-1.x` which we can change to `version: 1.x`

Comment (rayandas) [reply package/cfg/rke-cis-1.23-permissive/etcd.yaml:3]: Or we can keep all the configs as `version: cis-1.x` same as kube-bench. I shouldn’t impact anything.

**Review by andypitcher — APPROVED** [review]

**Review by rayandas — APPROVED** [review]

**Comment by andypitcher** [comment]
@rayandas we will make another PR to include @KiranBodipi's changes and  adjust all yamls to have `version: cis-1.x`.

File: package/cfg/rke-cis-1.23-permissive/etcd.yaml (hunk 1 above)
Comment (rayandas) [reply package/cfg/rke-cis-1.23-permissive/etcd.yaml:3]: Raised https://github.com/rancher/cis-operator/issues/248 to track the changes. Someone from my team will rake it up.

**Pull request merged**